client/blocks/reader-share/index.jsx

```diff
@@ -198,48 +199,42 @@ class ReaderShare extends React.Component {
 					</span>
 				</span>,
 				this.state.showingMenu &&
-					( canShareToWordPress()
-						? <SitesPopover
-								key="menu"
-								header={
-									<div>
-										{ this.props.translate( 'Share on:' ) }
-									</div>
-								}
-								context={ this.refs && this.refs.shareButton }
-								visible={ this.state.showingMenu }
-								groups={ true }
-								onSiteSelect={ this.pickSiteToShareTo }
-								onClose={ this.closeMenu }
-								position={ this.props.position }
-								className="reader-share__sites-popover"
-							/>
-						: <ReaderPopoverMenu
-								key="menu"
-								context={ this.refs && this.refs.shareButton }
-								isVisible={ this.state.showingMenu }
-								onClose={ this.closeExternalShareMenu }
-								position={ this.props.position }
-								className="popover reader-share__popover"
-								popoverTitle={ translate( 'Share on' ) }
-							>
-								<PopoverMenuItem
-									action="twitter"
-									className="reader-share__popover-item"
-									title={ translate( 'Share on Twitter' ) }
-								>
-									<SocialLogo icon="twitter" />
-									<span>Twitter</span>
-								</PopoverMenuItem>
-								<PopoverMenuItem
-									action="facebook"
-									className="reader-share__popover-item"
-									title={ translate( 'Share on Facebook' ) }
-								>
-									<SocialLogo icon="facebook" />
-									<span>Facebook</span>
-								</PopoverMenuItem>
-							</ReaderPopoverMenu> ),
+					<ReaderPopoverMenu
+						key="menu"
+						context={ this.refs && this.refs.shareButton }
+						isVisible={ this.state.showingMenu }
+						onClose={ this.closeExternalShareMenu }
+						position={ this.props.position }
+						className="popover reader-share__popover"
+						popoverTitle={ translate( 'Share on' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Share on:' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/integrate-sites-list-in-new-share-popover)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).